tests/features/maintenance.feature

```diff
@@ -23,8 +23,10 @@ Feature: Check maintenance mode
         """
         Then <lock_type> "<lock_host>" has holder "pgconsul_postgresql1_1.pgconsul_pgconsul_net" for lock "/pgconsul/postgresql/leader"
         When we set value "enable" for key "/pgconsul/postgresql/maintenance" in <lock_type> "<lock_host>"
+        And we wait "10.0" seconds
```

Comment:
Here and below we should wait til pgconsul's at all hosts read the maintenance node